packages/genesys-spark-components/src/components/stable/gux-icon/gux-icon.types.ts

```diff
@@ -273,6 +277,7 @@ export type GuxIconIconName =
   | 'fa/list-radio-regular'
   | 'fa/list-regular'
   | 'fa/list-ul-regular'
+  | 'fa/location-dot-slash-regular'
```

Comment:
These have to be added to the examples too right?

Reply:
Of course, sorry, I totally forgot about that file.